bin/wdq-batch

```diff
@@ -137,17 +146,25 @@ job.add_condor_cmd('getenv', 'True')
 job.add_condor_cmd('accounting_group', args.condor_accounting_group)
 job.add_condor_cmd('accounting_group_user', args.condor_accounting_group_user)
 if args.universe != 'local':
-    job.add_condor_cmd('request_memory', 4096)
+    if 'wpipeline' in args.wpipeline:
+        job.add_condor_cmd('request_memory', 4096)
+    else:
+        job.add_condor_cmd('request_memory', 32768)
 
 # add common wdq options
 job.add_opt('wpipeline', args.wpipeline)
 job.add_opt('colormap', args.colormap)
 job.add_opt('ifo', args.ifo)
-job.add_opt('condor', '')
 if args.config_file is not None:
     job.add_opt('config-file', args.config_file)
-if args.cache_file is not None:
-    job.add_opt('cache-file', args.cache_file)
+if 'wpipeline' in args.wpipeline:
+    job.add_opt('condor', '')
+    if args.cache_file is not None:
+        job.add_opt('cache-file', args.cache_file)
+else:
+    job.add_opt('far-threshold', str(args.far_threshold))
+    job.add_opt('nproc', str(args.nproc))
+    job.add_opt('verbose', '')
```

Comment:
I think we want instead:
```suggestion
    if args.verbose:
        job.add_opt('verbose', '')
```

Reply:
Good suggestion -- otherwise verbose output is always on!